src/grammar/grammar.gg

```diff
@@ -206,7 +206,7 @@ StatementBlock      = body:statements;
 StatementReturn     = keyword<"return"> expression:expression? semicolon;
 StatementExpression = expression:expression semicolon;
 StatementAssign     = left:expression operator:(augmentedOp / "=") right:expression semicolon;
-StatementCondition  = keyword<"if"> condition:expression trueBranch:statements falseBranch:(keyword<"else"> @(FalseBranch / StatementCondition))?;
+StatementCondition  = keyword<"if"> condition:expression? trueBranch:statements falseBranch:(keyword<"else"> @(FalseBranch / StatementCondition))?;
```

Comment:
let's add a comment explaining why the condition is optional in in the grammar, otherwise it will be confusing for new contributors